services/application-mapper/application-question.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+const AboutApplicationFormFields = require('../../constants/form-fields/about-application-form-fields');
+const CrimeFormFields = require('../../constants/form-fields/crime-form-fields');
+const InjuriesFormFields = require('../../constants/form-fields/injuries-form-fields');
+const PregnancyFormFields = require('../../constants/form-fields/pregnancy-form-fields');
+const ImpactFormFields = require('../../constants/form-fields/impact-form-fields');
+const ApplicantDetailsFormFields = require('../../constants/form-fields/applicant-details-form-fields');
+const MentalHealthFormFields = require('../../constants/form-fields/mental-health-form-fields');
+const OffenderFormFields = require('../../constants/form-fields/offender-form-fields');
+const TreatmentFormFields = require('../../constants/form-fields/treatment-form-fields');
+const OtherCompensationFormFields = require('../../constants/form-fields/other-compensation-form-fields');
+const mapTimestampToTariffFormat = require('./date-part-mapper');
+
+module.exports = class ApplicationQuestion {
```

Comment:
It's a class because for each question id coming in, we create an ApplicationQuestion object then set the columnValue and columnName so that all information needed about a question is contained within one class. If we need other information about the question later down the flow, such as the theme or question section then we have it on hand.

Reply:
We can maybe have a further discussion about his as a group before it gets signed off. I understand the pattern and how you've approached it, it'd make total sense in a class-based language but it's not the right approach for a JS app. In fact, the linter is supposed to prevent you from creating classes - you shouldn't have been able to commit this - but something's obviously not right in the config.